Resources/Prototypes/Entities/Markers/Spawners/Random/smokables.yml

```diff
@@ -0,0 +1,75 @@
+- type: entity
+  id: RandomSmokables
+  name: random smokable spawner
+  parent: MarkerBase
+  placement:
+    mode: PlaceFree
+  components:
+  - type: Transform
+    anchored: false
+  - type: Sprite
+    layers:
+      - sprite: Objects/Consumable/Smokeables/Cigarettes/Packs/black.rsi
+        state: closed
+  - type: RandomSpawner
+    prototypes:
+      - Cigarette
+      - Cigarette
+      - Cigarette
+      - Cigar
+      - CigPackGreen
+      - CigPackRed
+      - CigPackBlue
+      - CigPackBlack
+      - PackPaperRollingFilters
```

Comment:
Use the new tables because copy-pasting is awful.

Reply:
info about usage here:
https://docs.spacestation14.com/en/space-station-14/core-tech/entity-tables.html

Reply:
Ok thanks, converted to entity tables